src/modules/MeasureTool/MeasureToolCore/ScreenCapturing.cpp

```diff
@@ -197,13 +197,9 @@ std::unique_ptr<D3DCaptureState> D3DCaptureState::Create(DxgiAPI* dxgiAPI,
                                                                                             swapChain.put()));
 
     // We must create the object in a heap, since we need to pin it in memory to receive callbacks
-    auto statePtr = new D3DCaptureState{ dxgiAPI,
-                                         std::move(swapChain),
-                                         pixelFormat,
-                                         std::move(monitorInfo),
-                                         continuousCapture };
+    auto statePtr = std::unique_ptr<D3DCaptureState>(new D3DCaptureState{ dxgiAPI, std::move(swapChain), pixelFormat, std::move(monitorInfo), continuousCapture });
```

Comment:
why not use make_unique then?

Reply:
+1 to use make_unique

Reply:
The constructor is private, so it cant use it.

Reply:
Maybe we need to suppress a rule about using "new" in the future, but I think it is clear than the using make unique with a friend class or inheritance or the private constructor  argument trick.